src/psyclone/alg_gen.py

```diff
@@ -146,7 +142,7 @@ def gen(self):
         return self._ast
 
 
-def adduse(location, name, only=None, funcnames=None):
+def _adduse(location, name, only=None, funcnames=None):
```

Comment:
This break building the dev guide:
```
...
AttributeError: module 'psyclone.alg_gen' has no attribute 'adduse'
psyclone/doc/developer_guide$ grep alg_gen.adduse *rst
modules.rst:.. autofunction:: psyclone.alg_gen.adduse

Reply:
Good spot. Removed the offending reference and adding a warning note that the days of f2pygen are numbered.

Reply:
Great, thanks!